Plugins/JarImporterPlugin/JarImporterPlugin.swift

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import PackagePlugin
+
+@main
+struct JarImporterPlugin: CommandPlugin {
+  func performCommand(
+    context: PluginContext,
+    arguments: [String]
+  ) throws {
+    guard arguments.count == 2,
+          let mouduleName = arguments.first,
+          let jarFile = arguments.last else {
+      print(
+        """
+        Please specify the module name and the Java jar file.
+        Usage: swift package import-java-jar MySwiftTarget MyJavaLib.jar
+        """)
+      return
+    }
+    let jarImporter = try context.tool(named: "JarImporter")
+    let sometoolArgs = [
+      "--module-name", mouduleName,
+      "--jar-file", jarFile,
```

Comment:
```suggestion
      "--jar", jarFile,
```

please :)

Reply:
I did this on purpose in order to differentiate from the `jar` usage in the `JavaToSwift`. I will think about it and update :)